pages/common/vlc.md

```diff
@@ -0,0 +1,17 @@
+# vlc
+
+> VLC media player.
+> Cross-platform multimedia player.
+> More information: <https://www.videolan.org/vlc/>.
+
+- Play an audio or video file:
```

Comment:
```suggestion
- Execute a multimedia file:
```

Reply:
```suggestion
- Play a file:
```

Reply:
Maybe?
```suggestion
- Play a multimedia file:
```

Reply:
Sure! But isn't it clear that a file that is played by a media player should be a multimedia file?
However, I feel like, the change is not significant enough to be worth arguing about 😄.